open_instruct/dataset_transformation.py

```diff
@@ -1255,6 +1296,12 @@ def rlvr_filter_v1(
     return max_prompt_token_length_ok and max_token_length_ok and (contain_some_labels or not need_contain_labels)
 
 
+def rlvr_filter_v2(row: Dict[str, Any], tokenizer: PreTrainedTokenizer, max_prompt_token_length: Optional[int] = None):
```

Comment:
Can we call this "max_length_filter"?

Reply:
I think the tricky thing is that SFT/DPO use a different filter, since they take into account full sample length (max_token_length) instead of just prompt (max_prompt_token_length)?